### PR TITLE
Add new file: skateboard

### PR DESCRIPTION
**Pull request by Lauramirez-2ii**

### CODE REVIEW

File: R/skateboard_Laura.Rmd [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Skateboard
+output: github_document
```
Comment (maurolepore) [inline R/skateboard_Laura.Rmd:3]: Awesome!
Can you add this?
author: Laura

Comment (Lauramirez-2ii) [reply R/skateboard_Laura.Rmd:3]: where should I add it ?

Comment (maurolepore) [reply R/skateboard_Laura.Rmd:3]: Thanks

**Review by maurolepore — APPROVED** [review]
Congratulations!

**Pull request merged**